tests/resources/test_gemd_resource.py

```diff
@@ -0,0 +1,85 @@
+import uuid
+import random
+
+import pytest
+
+from citrine.resources.data_concepts import DataConcepts
+from citrine.resources.gemd_resource import GEMDResourceCollection
+from citrine.resources.material_run import MaterialRun
+from citrine.resources.material_spec import MaterialSpec
+
+from tests.utils.factories import MaterialRunDataFactory, MaterialSpecDataFactory
+from tests.utils.session import FakeSession, FakeCall
+
+
+@pytest.fixture
+def session() -> FakeSession:
+    return FakeSession()
+
+
+@pytest.fixture
+def collection(session) -> GEMDResourceCollection:
+    return GEMDResourceCollection(
+        project_id=uuid.uuid4(),
+        dataset_id=uuid.uuid4(),
+        session=session
+    )
+
+
+def sample_gems(nsamples, **kwargs):
+    factories = [MaterialRunDataFactory, MaterialSpecDataFactory]
+    return [random.choice(factories)(**kwargs) for _ in range(nsamples)]
+
+
+def test_gemd(collection, session):
+    # Given
+    samples = sample_gems(20)
+    session.set_response({
+        'contents': samples
+    })
+
+    # When
+    gems = list(collection.list())
+
+    # Then
+    assert 1 == session.num_calls
+    expected_call = FakeCall(
+        method='GET',
+        path='projects/{}/storables'.format(collection.project_id),
```

Comment:
Using string formatting methods to assemble URLs isn't great practice (see #650), but it's not worth sweating since this is only a test.  Any reason you didn't use an `f""` string?
```suggestion
        path=format_escaped_url('projects/{}/storables', collection.project_id),
```

Reply:
easy fix, thanks for pointing out. and yes, all implementations were passed to DataConceptsCollection after switching inheritance, so should be consistent.